test/rest.py

```diff
@@ -29,7 +29,13 @@
 @author: Jens Timmerman (Ghent University)
 """
 import os
-from urllib2 import HTTPError
+
+try:
+    # Python 3
+    from urllib.request import HTTPError
```

Comment:
add it to the py23 module, we can use this in other places too

Reply:
fixed